hack/packages/verify-openapischema-for-package.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# this shell script checks the existence of openAPIv3 schema in package before pushing the imgpkg bundle for package.
+
+# set this value to your package name
+PACKAGE=$1
+
+# set this value to the version of your package
+VERSION=$2
+
+if [ -z "$PACKAGE" ]
+then
+  echo "check sample values for package failed. must set PACKAGE"
+  exit 2
+fi
+
+if [ -z "$VERSION" ]
+then
+  echo "check sample values for package failed. must set VERSION"
+  exit 2
+fi
+
+ROOT_DIR="addons/packages"
+PACKAGE_DIR="${ROOT_DIR}/${PACKAGE}"
+VERSION_DIR="${PACKAGE_DIR}/${VERSION}"
+ARTIFACTS_DIR="${VERSION_DIR}/artifacts"
+BUNDLE_DIR="${VERSION_DIR}/bundle"
+CONFIG_DIR="${BUNDLE_DIR}/config"
+RED='\033[0;31m'
+YELLOW='\033[0;33m'
+NC='\033[0m' # No Color
+GREEN='\033[0;32m'
+
+verify_openapischema_for_package() {
+  schema_file="${CONFIG_DIR}/schema.yaml"
+  if [ -f "${schema_file}" ]; then
+    mkdir -p "${ARTIFACTS_DIR}"
+    cd "${ARTIFACTS_DIR}" || exit
+    ytt -f ../bundle/config/schema.yaml --data-values-schema-inspect -o openapi-v3 > generated-openapi-schema.yaml
+    yq e '.components.schemas.dataValues' generated-openapi-schema.yaml > schema-contents.yaml
```

Comment:
(comment): Since we introducing `yq` and `kapp` into the dev process it would be good to create tool targets for these that ensure install with appropriate versions (similar to downstream).

Reply:
Discussed with @shivaani0505 about this before. TCE doesn't have such make targets to install tools. They directly use the binaries installed in the users' path. We probably need TCE folks to chime in if we decide to add these tool make targets.

Reply:
I had suggested the same in the link https://github.com/vmware-tanzu/community-edition/pull/2719#issuecomment-1006976852, and also discussed with @seemiller . He said he would bring it up in TCE meeting

Reply:
Sounds good.